rae_hw/src/peripherals/speakers.cpp

```diff
@@ -83,8 +104,8 @@ int main(int argc, char* argv[]) {
     rclcpp::init(argc, argv);
 
     auto node = std::make_shared<rae_hw::SpeakersNode>(rclcpp::NodeOptions());
-    rclcpp::executors::StaticSingleThreadedExecutor executor;
-    executor.add_node(node);
+    rclcpp::executors::SingleThreadedExecutor executor;
```

Comment:
Any reason why a dynamic single threaded executor used instead of static one?

Reply:
Updated to static